SOMObject obj1
Relation relation
SOMObject obj2
